### PR TITLE
fix: Now error is being removed when new request is ok

### LINKED ISSUE #2449: Bug: warning on highlight isn't removed when fixed
### Describe the bug

I pasted in the full repo address for the intro course, which caused the error to display. However, once I updated to the correct format (open-sauced/intro), the error remained.

image

### Steps to reproduce

1. Go to highlights
2. create a new highlight
3. in the repo, put a complete URL (https://github.com/open-sauced/app/)
5. change the repo to open-sauced/app
6. Continue to see the error

### Browsers

Chrome

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by aryanas159**
## Description

fixes #2449

I just added a ```setError("")``` in case there is a new API request which responds with 200.
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by BekahHW** [comment]
Hey @aryanas159 can you add a video to show that the code is working?

**Comment by aryanas159** [comment]
> Hey @aryanas159 can you add a video to show that the code is working?

Sure @BekahHW,
Here it is.

https://github.com/open-sauced/app/assets/114330931/18b34808-46eb-42cf-8856-3aac50c8a880

**Review by ThePiyushAggarwal — COMMENTED** [review]
Looks good to me 👍

**Review by NsdHSO — APPROVED** [review]
LGTM

File: components/molecules/HighlightInput/highlight-input-form.tsx [hunk 1]
```diff
@@ -355,7 +355,7 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
       setError("Repo not found!");
       return;
     }
-
+    setError("");
```
Comment (nickytonline) [inline components/molecules/HighlightInput/highlight-input-form.tsx:358]: nit: move `setError("")` to he beginning of the function.

**Review by nickytonline — APPROVED** [review]
Thanks for making the change @aryanas159. Left a comment, but it's working as expected.

**Pull request merged**